scripts/build_android.bat

```diff
@@ -0,0 +1,80 @@
+@echo off
+setlocal
+
+:: =============================
+:: Configuration
+:: =============================
+set "APP_NAME=%~1"
+if "%APP_NAME%"=="" (
+    echo [ERROR] Application name argument missing.
+    exit /b 1
+)
+
+set "QT_ROOT=C:\Qt6.8.2\6.8.2"
+set "QT_ANDROID_BIN=%QT_ROOT%\android_arm64_v8a\bin"
+set "QT_DEPLOY_TOOL=%QT_ROOT%\mingw_64\bin\androiddeployqt.exe"
+set "MAKE_EXE=%USERPROFILE%\AppData\Local\Android\Sdk\ndk\28.0.12433566\prebuilt\windows-x86_64\bin\make.exe"
+set "ANDROID_NDK_ROOT=%USERPROFILE%\AppData\Local\Android\Sdk\ndk\28.0.12433566"
+set "ANDROID_PLATFORM=android-34"
```

Comment:
There are some assumptions here. Unclear if this works for everyone, or if the DSA team has standardized around these paths.